### PR TITLE
Topic 1

### PR DESCRIPTION
**Pull request by aggomez**

### CODE REVIEW

File: 01-js-intro-and-jquery/spotify.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+	var searchList = function(query){
+		$.ajax({
+			type: "GET",
+			url: "https://api.spotify.com/v1/search",
+			data: { q: query, type: 'track' },
+			success: function(response){
+				$("ul").empty();
+				console.log(response);
+				if(response.tracks.items.length > 0){
```
Comment (troyax) [inline 01-js-intro-and-jquery/spotify.js]: This line is the same as 

```
if (response.tracks.items.length) { ... }
```

If the length is 0 will be falsy so this want be executed

File: 01-js-intro-and-jquery/spotify.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+$(document).ready(function(){
+	
+	$("input").focus();
+
+	var searchList = function(query){
```
Comment (troyax) [inline 01-js-intro-and-jquery/spotify.js]: Try to write the functions definitions as

```
var sarasaFunction = function (...) { 
  ... 
};
```

Just for conventions and good reading of the code

File: 01-js-intro-and-jquery/spotify.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+						var trackname = response.tracks.items[index].name;
+						var artist = response.tracks.items[index].artists[0].name;
+						var duration = response.tracks.items[index].duration_ms;
+						$(".songs").append("<li> Track: " +trackname+ "<br> Artist: " +artist+ "<br>Duration: " +duration+ "</li>");
+					})
+					$(".hidden").fadeIn("slow");
+				};
+			}
+		});
+	}
```
Comment (troyax) [inline 01-js-intro-and-jquery/spotify.js]: When defining a variable you must use `;` at the end of the line

**Comment by troyax** [comment]
Just a few comments, good to go in my opinion

File: 01-js-intro-and-jquery/spotify.js (hunk 1 above)
Comment (aggomez) [inline 01-js-intro-and-jquery/spotify.js]: noted!

File: 01-js-intro-and-jquery/spotify.js (hunk 3 above)
Comment (aggomez) [inline 01-js-intro-and-jquery/spotify.js]: Missed that one. Fixing it before merging.

File: 01-js-intro-and-jquery/spotify.js (hunk 2 above)
Comment (aggomez) [inline 01-js-intro-and-jquery/spotify.js]: noted!

**Pull request merged**